plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -46,6 +50,12 @@ const sampleConfig = `
 #  "Server=192.168.1.10;Port=1433;User Id=<user>;Password=<pw>;app name=telegraf;log=1;",
 # ]
 
+## This enables a specific set of queries depending on the database type. It replaces azuredb = true/false.
+## In the config file, the sql server plugin section can be repeated multiple times each with a set of servers for a specific database_type
+## possible values for database_type are  "AzureSQLDB" or  "SQLServer" or "AzureSQLManagedInstance"
```

Comment:
as in the docs, a list is easier to read and identify
```
## possible values are
## - SQLServer
## - AzureSQLDB
## - AzureSQLManagedInstance
```

Reply:
Done